MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+</tr>
+
+<tr>
+<td>{{'[ErrorMessageColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ErrorMessageColor)'| markdownify }}</td>
+<td>ErrorLabelTextStyle</td>
+<td>Gets or sets the style of error message label , that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>{{'[ValidationLabelStyle](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ValidationLabelStyle)'| markdownify }}</td>
+<td>ValidMessageLabelTextStyle</td>
+<td>Gets or sets the style of valid message label, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>LabelTextStyle</td>
+<td>Gets or sets the style of editor label text, that used to customize the text color, font, font size, font family and font attributes.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>Background</td>
+<td>Gets or sets the background to the data form item layout which is used to render label and editor for the data field.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>DefaultLayoutSettings</td>
+<td>Gets or sets the value of layout (holds label and editor) settings which is used to customize the label position, label width and editor width.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>ItemsOrderInRow</td>
+<td>Gets or sets the editor’s position in a row.</td>
+</tr>
+
+<tr>
+<td>{{'[ImageSource](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ImageSource)'| markdownify }}</td>
+<td>LeadingLabelIcon</td>
+<td>Gets or sets an image used as the editor’s label.</td>
+</tr>
+
+
+<tr>
+<td>{{'[IsValid](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_IsValid)'| markdownify }}</td>
+<td>IsValid</td>
+<td>Gets a value indicating whether this DataFormItem is valid or not.</td>
+</tr>
+
+<tr>
+<td>{{'[PropertyInfo](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_PropertyInfo)'| markdownify }}</td>
+<td>PropertyInfo</td>
+<td>Gets the property info of the data field.</td>
+</tr>
+</table>
+
+### DataFormDefaultLayoutSettings
+
+Following properties of `Xamarin SfDataForm` is moved to `DataFormDefaultLayoutSettings` class of `.NET MAUI SfDataForm`.
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th>
+</tr>
+
+<tr>
+<td>{{'[LabelPosition](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.SfDataForm.html#Syncfusion_XForms_DataForm_SfDataForm_LabelPosition)'| markdownify }}</td>
+<td>LabelPosition</td>
+<td>Gets or sets the label position for DataFormItem to define the position of the label which associated with the editor.</td>
```

Comment:
<td>Gets or sets the label position for DataFormItem to define the position of the label associated with the editor.</td>